### PR TITLE
OKTA-620015 Update "Upgrade SAML Certs" guide

### PR DESCRIPTION
**Pull request by danielmaharry-okta**
## Description:
- **What's changed?** 
- New Admin Console sections to [/docs/guides/updating-saml-cert](https://developer.okta.com/docs/guides/updating-saml-cert/main/)
- Reworked old API section for clarity

07/28. Preview at https://64c3ee2327ebb725a6a4fa6c--reverent-murdock-829d24.netlify.app/docs/guides/updating-saml-cert/main/

- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-620015](https://oktainc.atlassian.net/browse/OKTA-620015)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md [hunk 1]
```diff
@@ -4,206 +4,172 @@ excerpt: Upgrade SAML Apps to SHA256
...
-### Existing SAML 2.0 app integrations
+      ![The SAML Signing Certificates section of the Applications UI](/img/updating-saml-certs/signing-certificates-ui.png)
 
-To update existing app integrations, you first need to list your apps and get the app id, name, and label for each app to update. Then for each app to update, perform the following steps:
+   </div>
 
-1. Generate a new application key credential.
-1. Update the key credential for the app to specify the new signing key id.
-1. Upload the new certificate to the ISV (this step cannot be automated.)
+1. If a certificate of **Type** SHA-2 is active, you don’t need to upgrade the certificate.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md:43]: Should these two items be in an unordered list beneath the action step preceding them?
```suggestion
* If a certificate of **Type** SHA-2 is active, you don’t need to upgrade the certificate.
```

File: packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md [hunk 2]
```diff
@@ -4,206 +4,172 @@ excerpt: Upgrade SAML Apps to SHA256
...
+1. If a certificate of **Type** SHA-1 is active, you need to create a SHA2 certificate and make it active.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md:44]:
```suggestion
  * If a certificate of **Type** SHA-1 is active, you need to create a SHA2 certificate and make it active.
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM

Nice work. Just a small formatting suggestion.

**Comment by danielmaharry-okta** [comment]
Thanks @brentschaus-okta . Fixed the nits. Just waiting for a second tech review before merging it.

**Comment by danielmaharry-okta** [comment]
Approved by Jeff Taylor

**Pull request merged**